src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,71 @@
+import { useContext } from 'react';
+import cn from 'classnames';
+import { Filter } from '../../services/enums';
+import { TodosContext } from '../../TodosContext';
+import { Todo } from '../../types';
+
+function countActiveTodos(todos: Todo[]): number {
+  return todos.filter(todo => {
+    return !todo.completed;
+  }).length;
+}
+
+export const TodoFooter: React.FC = () => {
+  const {
+    todos,
+    clearAllCompleted,
+    filterBy,
+    setFilterBy,
+    isTodosHasCompleted,
+  } = useContext(TodosContext);
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${countActiveTodos(todos)} `}
+        items left
+      </span>
+
+      <nav className="filter">
+        <a
+          href="#/"
+          className={cn('filter__link', {
+            selected: filterBy === Filter.ALL,
+          })}
+          onClick={() => setFilterBy(Filter.ALL)}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={cn('filter__link', {
+            selected: filterBy === Filter.ACTIVE,
+          })}
+          onClick={() => setFilterBy(Filter.ACTIVE)}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={cn('filter__link', {
+            selected: filterBy === Filter.COMPLETED,
+          })}
+          onClick={() => setFilterBy(Filter.COMPLETED)}
+        >
+          Completed
+        </a>
+      </nav>
+
+      <button
+        type="button"
+        className="todoapp__clear-completed"
+        onClick={clearAllCompleted}
+        style={{ opacity: isTodosHasCompleted() ? 1 : 0 }}
```

Comment:
Please use display: none; or unmount this button from DOM, because now I can click on it
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/54936044/67dab5ae-808d-4367-8c12-8d3a62e42dd7)

Reply:
used not display: none but pointerEvents: none to not move filters to right side